utils/build.ps1

```diff
@@ -1606,6 +1608,52 @@ function Build-LLVM([Platform]$Platform, $Arch) {
     }
 }
 
+function Build-Sanitizers([Platform]$Platform, $Arch) {
+  $BareTarget = $Arch.LLVMTarget.Replace("$AndroidAPILevel", "")
+  $LLVMDir = "$(Get-TargetProjectBinaryCache $Arch LLVM)\lib\cmake\llvm"
```

Comment:
Should this be target? What happens if we don't have a target build of LLVM? (e.g. Android)

Reply:
This came up as a warning during configuration. In many cases standalone builds reach out to LLVM for configuration details or test dependencies. That should work because we do configure LLVM for Android targets. Let me double-check that.

Reply:
Done. It's required for testing and otherwise silences this warning: https://github.com/llvm/llvm-project/blob/57b48987f6c21e369e7bb1626dc79ca74aa34fdb/compiler-rt/cmake/Modules/CompilerRTUtils.cmake#L314